sdformat_urdf/test/link_tests.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <gtest/gtest.h>
+#include <sdf/sdf.hh>
+#include <sdformat_urdf/sdformat_urdf.hpp>
+#include <urdf_model/model.h>
+#include <urdf_model/types.h>
+
+#include "sdf_paths.hpp"
+#include "test_tools.hpp"
+
+TEST(Link, link_inertia)
+{
+  sdf::Errors errors;
+  urdf::ModelInterfaceSharedPtr model = sdformat_urdf::parse(
+    get_file(PATH_TO_SDF_LINK_INERTIA), errors);
+  EXPECT_TRUE(errors.empty()) << errors;
+  ASSERT_TRUE(model);
+  ASSERT_EQ("link_inertia", model->getName());
+
+  urdf::LinkConstSharedPtr link = model->getLink("link");
```

Comment:
Could you add an assert for `link` here and in the other tests in this file?

Reply:
Added assert's for link not being null in ec3f81b